<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Parser</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="ignoreBrCheckBox">
          <property name="text">
           <string>Ignore &lt;Br&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="resolvePostContentCheckBox">
          <property name="text">
           <string>Generate Post Content</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ignoreClosingTagsCheckBox">
          <property name="text">
           <string>Ignore closing tags</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="collapseAttributesArrayCheckBox">
          <property name="text">
           <string>Collaspe attributes into Array</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Generator</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="wrapInClassCheckBox">
          <property name="text">
           <string>Wrap In Class</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="classEdit">
          <property name="placeholderText">
           <string>class name</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Expression</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Type ID Expression (%tid%)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="typeIDedit">
          <property name="text">
           <string>%tag%</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Variable Name Expression (%vname%)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="variableNameEdit">
          <property name="text">
           <string>%id%</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Empty Name Default</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="variableNameEdit_2">
          <property name="text">
           <string>%tag%_%tid%</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>Auto ID attribute Expression</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="autoIDEdit">
          <property name="text">
           <string>%vname%</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
